tests/platform/check_critical_services.py

```diff
@@ -9,19 +9,29 @@
 from common.utilities import wait_until
 
 
+def _all_critical_services_fully_started(dut):
+    logging.info("Check critical service status")
+    if not dut.critical_services_fully_started():
+        logging.info("dut.critical_services_fully_started is False")
```

Comment:
message to the user should be friendly enough instead of local parameter. 
could it be that the list of critical services not up will be provided as info to the user for better debuggability? maybe even provide status of what is running and what is not?

Reply:
Hi Liat,
This has been addressed in another PR [#1395](https://github.com/Azure/sonic-mgmt/pull/1395) by promoting the level of log which outputs the services’ state to “info”. 
By doing so it’s able to give the user clear information.